core/geometry/src/polyface/PolyfaceQuery.ts

```diff
@@ -431,95 +483,104 @@ export class PolyfaceQuery {
   }
   /**
    * Test for convex volume by dihedral angle tests on all edges.
-   * * This tests if all dihedral angles are positive.
+   * * This tests if all dihedral angles are positive. This means the mesh is probably convex with outward normals.
    * * In a closed solid, this is a strong test for overall convexity.
-   * * With `ignoreBoundaries` true, this may be a useful test when all the facets are in a single edge-connected component, such as a pyramid with no underside.
+   * * With `ignoreBoundaries` true, this may be a useful test when all the facets are in a single edge-connected
+   * component, such as a pyramid with no underside.
    * * It is not a correct test if there are multiple, disjoint components.
    *   * Take the above-mentioned pyramid with no underside.
    *   * Within the same mesh, have a second pyramid placed to the side, still facing upward.
    *   * The angles will pass the dihedral convexity test, but the composite thing surely is not convex.
-   * @param source mesh to examine
-   * @param ignoreBoundaries if true, ignore simple boundary edges, i.e. allow unclosed meshes.
-   * @returns true if the mesh is closed and has all dihedral angles (angle across edge) positive
+   * @param source mesh.
+   * @param ignoreBoundaries if `true` ignore simple boundary edges, i.e., allow unclosed meshes. Default is `false`.
+   * @returns true if all dihedral angles (angle across facets) of the mesh are positive. The mesh is probably convex
+   * with outward normals.
    */
   public static isConvexByDihedralAngleCount(source: Polyface, ignoreBoundaries: boolean = false): boolean {
     return this.dihedralAngleSummary(source, ignoreBoundaries) > 0;
   }
   /**
   * Compute a number summarizing the dihedral angles in the mesh.
-  * @see [[isConvexByDihedralAngleCount]] for comments about ignoreBoundaries===true when there are multiple connected components.
-  * @param source mesh to examine
-  * @param ignoreBoundaries if true, ignore simple boundary edges, i.e. allow unclosed meshes.
+  * @param source mesh.
+  * @param ignoreBoundaries if `true` ignore simple boundary edges, i.e., allow unclosed meshes. Default is `false`.
   * @returns a number summarizing the dihedral angles in the mesh.
```

Comment:
We changed the bahaviour of "dihedralAngleSummary"
It is explained in the NextVersion.md file